src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.container.js

```diff
@@ -16,6 +18,11 @@ export const EnrollmentRegistrationEntry: ComponentType<OwnProps> = ({
     const orgUnitId = useCurrentOrgUnitInfo().id;
     const { orgUnit, error } = useRulesEngineOrgUnit(orgUnitId);
     const { teiId, ready, skipDuplicateCheck } = useLifecycle(selectedScopeId, id, trackedEntityInstanceAttributes, orgUnit);
+    const isUserInteractionInProgress: boolean = useSelector(
+        state =>
+            dataEntryHasChanges(state, 'newPageDataEntryId-newEnrollment')
+          || dataEntryHasChanges(state, 'newPageDataEntryId-newTei'),
```

Comment:
Hey, 
How about the case when creating a new TEI/enrollment to link as a relationship to an event? I believe these use cases are covered by adding checks for:

- `dataEntryHasChanges(state, 'relationship-newTei')`
- `dataEntryHasChanges(state, 'relationship-newEnrollment')`

<img width="748" alt="Screenshot 2023-01-03 at 09 16 59" src="https://user-images.githubusercontent.com/6817073/210321535-478b7094-0ffc-42f3-95cf-74cb5421cbb3.png">

 WDYT? Thank you!

Reply:
hey @simonadomnisoru, good catch! I updated the code to reflex the changes. Thanks